### DISCUSSION TITLE
Welcome to Harvest Discussions!

### DISCUSSION THREAD

**Discussion by cgrinds** [NetApp/harvest#579] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other Harvest community members. We hope that you:
  * Ask questions you’re wondering about
  * Share ideas
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dudy-duce** [comment]
Hello Harvest developpers & community,

This new version do not provide graphite compatibility like older versions ?

**Reply by cgrinds** [reply]
Hi @figeac888 that's correct. Graphite is no longer used by Harvest.

**Comment by prgsing** [comment]
Can Harvest be used with software version of Ontap?

**Reply by cgrinds** [reply]
hi @prgsing yes, that should work. Let us know if you run into any problems when you try it, and we'll troubleshoot.

**Reply by prgsing** [reply]
Thank you for the confirmation. Providing details on the environment - Ontap is deployed as a software on AKS (azure kubernetes service) cluster. I was wondering if you have any idea on what the configuration would look like for a setup like that.

**Reply by cgrinds** [reply]
Thanks for the details. Can you share any documentation for how you setup ONTAP on AKS? Are you referring to https://learn.microsoft.com/en-us/azure/aks/azure-netapp-files?

Either way, you should be able to treat the ONTAP deployment separately and install Harvest on [another machine](https://netapp.github.io/harvest/nightly/install/native/) or [K8s](https://netapp.github.io/harvest/nightly/install/k8/) and then follow the [prepare cDOT](https://netapp.github.io/harvest/nightly/prepare-cdot-clusters/) and [Harvest setup](https://netapp.github.io/harvest/nightly/configure-harvest-basic/) steps to monitor ONTAP.

**Reply by prgsing** [reply]
https://learn.microsoft.com/en-us/azure/aks/azure-netapp-files is from workload usecase perspective. What I am talking about is from Infrastructure point of view

We are using Harvest with ANF (azure netapp files) already because the becakend is hardware. I am trying find out if we can use harvest for Atom where Atom is a software-defined variant of NetApp's Cloud Volumes. It runs a trimmed-down version of ONTAP that runs on Kubernetes and integrates with the cloud environment for networking and storage. I am afraid there is no public documentation on the architecture of Atom. We have some internal documents though
